CHANGELOG.md

```diff
@@ -1,5 +1,98 @@
 # Changelog for TGM Plugin Activation library
 
+## Unreleased
+
+This is a major update which brings some interesting new features and fixes tons of bugs. This version of TGMPA is brought to you by [Thomas Griffin] with graceful thanks to [Gary Jones] and our new core-team member [Juliette Reinders Folmer] for the majority of the changes.
```

Comment:
"The majority of the changes in this version of TGMPA has been completed by the new core-team member [Juliette Reinders Folmer] and reviewed by [Gary Jones]."

Let's tell it straight.